staging/src/k8s.io/apimachinery/pkg/util/strategicpatch/testing/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
```

Comment:
add `testonly = True` to make sure that test libraries don't get included in production code.